odl/set/sets.py

```diff
@@ -138,6 +148,16 @@ def contains_set(self, other):
         """Return `True` for the empty set, otherwise `False`."""
         return isinstance(other, EmptySet)
 
+    def contains_all(self, other):
+        """Return `True` for an empty sequence, `False` else."""
```

Comment:
How about a list of `None`'s? I'd remove this and use the default implementation, it still works:

```python
>>> O = odl.EmptySet()
>>> all([x in O for x in []])
True
>>> all([x in O for x in [None]])
True
>>> all([x in O for x in [None, 'not-none']])
False
```